InterfacePython/TasmanianSG.py

```diff
@@ -50,6 +50,33 @@
 lsTsgLocalRules = ["localp", "semi-localp", "localp-zero", "localp-boundary"]
 lsTsgAccelTypes = ["none", "cpu-blas", "gpu-default", "gpu-cublas", "gpu-cuda", "gpu-rocblas", "gpu-hip", "gpu-magma"]
 
+def getTsgLib(tasmanian_library=0):
```

Comment:
Putting the library inside the TasmanianSparseGrid class was an old thing and no longer needed, CMake and pip set the paths well enough so we don't have to do this trick of passing a custom path to the library.

Look at the more recent code inside TasmanianDreamState.py and TasmanianDreamSampler.py files and make the loaded library common to the module and not specific to the class.

Reply:
Done.